string num_bottles
---
bool success
string message
